src/tito/cli.py

```diff
@@ -267,8 +267,24 @@ def main(self, argv):
             if e.errno != errno.EEXIST:
                 raise
 
+    @property
+    def initial_config(self):
+        """
+        Configuration that is dumped into `.tito/tito.props`
+        """
+        config = RawConfigParser()
+        config.setdefault(BUILDCONFIG_SECTION, {
+            "builder": "tito.builder.Builder",
+            "tagger": "tito.tagger.VersionTagger",
+            "changelog_do_not_remove_cherrypick": 0,
+            "changelog_format": "%s (%ae)",
+        })
+        return config
+
     def load_config(self, package_name, build_dir, tag):
-        self.config = ConfigLoader(package_name, build_dir, tag).load()
+        self.config = self.initial_config
+        if not self.options.without_init:
+            self.config = ConfigLoader(package_name, build_dir, tag).load()
```

Comment:
I think this should be:
```
        if self.options.without_init:
            self.config = self.initial_config
        else:
            self.config = ConfigLoader(package_name, build_dir, tag).load()
```

And the error message of ConfigLoader() should be altered and when it prints:
```
ERROR: Unable to locate branch configuration: /home/msuchy/fedora/git/gnome-logs/rel-eng/tito.props
Please run 'tito init'
```
We should add a line:
```
or run tito --without-init
```

Reply:
Good points, thank you. Updated.